src/executables/commdb.c

```diff
@@ -704,7 +704,7 @@ process_is_registered_proc (CSS_CONN_ENTRY * conn, char *args)
   rid = send_request_one_arg (conn, IS_REGISTERED_HA_PROC, (char *) buffer, len);
   return_string (conn, rid, &reply_buffer, &size);
 
-  if (size > 0 && strcmp (reply_buffer, HA_REQUEST_SUCCESS) == 0)
+  if (size > 0 && strncmp (reply_buffer, HA_REQUEST_SUCCESS, size - 1) == 0)
```

Comment:
The reason to use "size-1" instead of "size" is to compare until the '\0'? I guess we don't need to do it and "size" looks more readable.

Reply:
It is a right reason what you said. Thanks for your feedback but I gonna not to change this code because someone can easily understand the size includes the length of the null termination.